### DISCUSSION TITLE
Nicer autocomplete for JSON property (automatically add double quotes)

### DISCUSSION THREAD

**Discussion by ghost** [Kong/insomnia#3329] [Ideas]
Is there a way to enable autocomplete double quotes for JSON properties, or create a custom ?
In VSCode if you write: `{ test }` and hit CTLR + Space in `test` it suggests `"test"` for completion in JSON files, simple yet useful when writing JSON request body.

Example:
![image](https://user-images.githubusercontent.com/23178554/116192887-80ee4380-a6fc-11eb-8b07-5fbebf3ae20e.png) ![image](https://user-images.githubusercontent.com/23178554/116192847-6ddb7380-a6fc-11eb-91e5-5d043a3d589b.png)

**Comment by wdawson** [comment]
Hey @RickStanley! Thanks for the question. I don't think it's possible today so I'm going to convert this to an Idea 💡 

Thanks for submitting!
